rancher_pr_deploy.sh

```diff
@@ -17,25 +23,35 @@ if [[ -n "$TRAVIS" ]]; then
   # IF PULL REQUEST
   if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then
 
-    install_rancher
+    if [ ! "$(command -v rancher)" ]
+    then 
+      install_rancher
+    fi
 
-    curl -s "https://raw.githubusercontent.com/nhsuk/nhsuk-rancher-templates/master/templates/${RANCHER_TEMPLATE_NAME}/0/docker-compose.yml"  -o docker-compose.yml
-    curl -s "https://raw.githubusercontent.com/nhsuk/nhsuk-rancher-templates/master/templates/${RANCHER_TEMPLATE_NAME}/0/rancher-compose.yml" -o rancher-compose.yml
+    curl -s "https://raw.githubusercontent.com/nhsuk/nhsuk-rancher-templates/feature/changes-for-ci-script/templates/${RANCHER_TEMPLATE_NAME}/0/docker-compose.yml"  -o docker-compose.yml
+    curl -s "https://raw.githubusercontent.com/nhsuk/nhsuk-rancher-templates/feature/changes-for-ci-script/templates/${RANCHER_TEMPLATE_NAME}/0/rancher-compose.yml" -o rancher-compose.yml
 
-    touch answers.txt
-    echo -n "" > answers.txt
+    # need two args for the split but ORG is not needed
+    # shellcheck disable=SC2034 
+    IFS=/ read -r ORG REPO <<< "${TRAVIS_REPO_SLUG}"
 
-    {
-      echo "traefik_domain=dev.c2s.nhschoices.net"
-      echo "profiles_docker_image_tag=pr-${TRAVIS_PULL_REQUEST}"
-      echo "splunk_hec_endpoint=https://splunk-collector.cloudapp.net:8088"
-      echo "splunk_hec_token=${SPLUNK_HEC_TOKEN}"
-      echo "hotjar_id=265857"
-      echo "google_id=UA-67365892-5"
-      echo "webtrends_id=dcs222rfg0jh2hpdaqwc2gmki_9r4q"
-    } >> answers.txt
+    cat <<EOT  > answers.txt
+traefik_domain=dev.c2s.nhschoices.net
+$(create_parameter_name "$REPO")_docker_image_tag=pr-${TRAVIS_PULL_REQUEST}
+splunk_hec_endpoint=https://splunk-collector.cloudapp.net:8088
+splunk_hec_token=${SPLUNK_HEC_TOKEN}
+hotjar_id=265857
```

Comment:
These values are hard coded here and also in the rancher-compose.yml as defaults. Probably better, from a code duplication perspective rather than secret hiding, for them to be env vars and defined in a single place.